.github/workflows/unit-tests.yml

```diff
@@ -11,9 +11,12 @@ jobs:
     if: github.repository == 'openedx/edx-platform' || github.repository == 'edx/edx-platform-private'
     runs-on: [ edx-platform-runner ]
     strategy:
+      fail-fast: false    # TEMP: will not be merged.
       matrix:
         python-version: ['3.8']
-        django-version: ["3.2"]
+        django-version:
+          - "pinned"
+          - "4.0"
```

Comment:
We should enable `Django 4.0` tests in a separate PR.

Reply:
Right, I had this here to show how it might be used, though these tests fail for now.  I can comment out this option, and just leave "pinned" as the only value in effect.